package.json

```diff
@@ -21,6 +21,8 @@
   },
   "dependencies": {
     "@react-spring/web": "^9.2.1",
+    "@spree/storefront-api-v2-sdk": "^4.12.0",
```

Comment:
Also I checked that this sdk has lodash as a dependency (which is pretty heavy), but its just being used once for the method `get`, same with the `qs` library that its used twice and same thing with `axios`, althought this last one may be used for different cases.

I point all of this as currently all dependencies are installed when doing an npm install (not all of them are used of course) and we try to avoid extra size at any cost

Reply:
@goncy 👋  We removed the `lodash` dependency.